packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -725,6 +785,45 @@ class ChoroplethMap extends React.Component<{
         )
     }
 
+    @computed get globeWidth(): number {
```

Comment:
Where is this used? I did a browser search for these in the PR screen and they didn't come up. Is something hooking in and getting them?